ee/acl/acl_test.go

```diff
@@ -2939,7 +2939,7 @@ func TestGuardianOnlyAccessForAdminEndpoints(t *testing.T) {
 					}`,
 			queryName:          "restore",
 			testGuardianAccess: true,
-			guardianErr:        "The path \"\" does not exist or it is inaccessible.",
+			guardianErr:        "The uri path: \"\" doesn't exist",
```

Comment:
NIT: Back-ticks, as used in the initialization of `guardianData` would make this easier to read. Also above on 2861.

Reply:
Done